common/handlers/groups/create.go

```diff
@@ -0,0 +1,66 @@
+package groups
+
+import (
+	"context"
+
+	dsw "github.com/aserto-dev/go-directory/aserto/directory/writer/v3"
+	"github.com/aserto-dev/scim/common/convert"
+	"github.com/aserto-dev/scim/common/model"
+	"github.com/elimity-com/scim"
+	serrors "github.com/elimity-com/scim/errors"
+)
+
+func (g GroupResourceHandler) Create(ctx context.Context, attributes scim.ResourceAttributes) (scim.Resource, error) {
+	groupName, ok := attributes["displayName"].(string)
+	if !ok {
+		return scim.Resource{}, serrors.ScimErrorInvalidSyntax
+	}
+
+	logger := g.logger.With().Str("method", "Create").Str("name", groupName).Logger()
+	logger.Info().Msg("create group")
+	logger.Trace().Any("attributes", attributes).Msg("creating group")
+
+	group := &model.Group{}
+
+	err := convert.Unmarshal(attributes, group)
+	if err != nil {
+		logger.Error().Err(err).Msg("failed to convert attributes to group")
```

Comment:
If `err != nil` then `logger.Err(err).Msg(...)` is sufficient. You don't need the extra `.Error()` call.
`logger.Err(err)` call `.Error()` if `err != nil` and `.Info()` otherwise.